config/userdata.yaml

```diff
@@ -0,0 +1,13 @@
+#cloud-config
+repo_update: true
+
+packages:
+  - docker 
+
+runcmd:
+  - [ sh, -c, echo "USERDATA BEGIN"]
+  - sudo service docker start
```

Comment:
Just wanted to note that this removes `yum update` and `yum install docker`.

Reply:
the `packages: docker` covers the latter.

Reply:
we could consider also adding `package_update: true`

Reply:
James Harrington confirmed in slack that we did not need to worry about doing `yum update`, so fine to leave `package_update: true` out fro now I think.